### ISSUE TITLE
Can not make the program function at all. I have no clue why.

### ISSUE THREAD

**Issue by boris-mandrovnyj-krizik** [Youcomposer/SkyAutoMusic#8]
To Whom It May Concern I am sorry to bother, but I am not able to make this work
I can not make this program work. After some time trying to fix this program I can not get it to work. I found out this program can not know on which disk it is, so i wrote into the code relocation to my C:\ where python is located. Even after this the app can not run and has always an issue. I tried to even redirect it exactly to the folder where is python that I use for my personal funky stuff in VSC to run my crappy calculator app. It still will not launch it self and it even says that the file I installed does not exist.
Yours Faithfully
![image](https://github.com/user-attachments/assets/5be26734-68c4-4214-ab1c-1b3515e6ab88)
![image](https://github.com/user-attachments/assets/79a8e25c-47ca-4db2-8771-0aaf91214ea0)
![image](https://github.com/user-attachments/assets/d7b989c3-50e8-4b82-a551-22b487973552)

**Comment by boris-mandrovnyj-krizik** [comment]
> does the error still exist? @boris-mandrovnyj-krizik 

Yes it still persist probably cause i didnt try fixing it afterwards.

**Comment by boris-mandrovnyj-krizik** [comment]
> https://github.com/OstbornTV/SkyAutoMusic_release/releases/download/0.3.4/SkyAutoMusic.exe
> 
> try my version of the autoplayer, 200 songs are included ^^
> 
> I have ‘improved’ @Youcomposer's version a little. you don't need Python on your PC, it's all included in the exe.

how much can i trust your source?

**Comment by boris-mandrovnyj-krizik** [comment]
> > > https://github.com/OstbornTV/SkyAutoMusic_release/releases/download/0.3.4/SkyAutoMusic.exe
> > > try my version of the autoplayer, 200 songs are included ^^
> > > I have ‘improved’ @Youcomposer's version a little. you don't need Python on your PC, it's all included in the exe.
> > 
> > 
> > how much can i trust your source?
> 
> A 10 out of 10, it is forbidden to distribute malicious code via GitHub. I can also release the source code. I haven't uploaded the code yet because I haven't got around to it yet, so for now only exe. I suggest I get back to you once I've uploaded the code.
> 
> But it's nice that there are still some who question it critically

I mean i dont want to sound harsh but i kinda want to be carefull.

**Comment by boris-mandrovnyj-krizik** [comment]
@OstbornTV you are a hero

**Comment by boris-mandrovnyj-krizik** [comment]
its fine i think i can tune the stuff, but I have atleasst the base yk. And yea i tried it and it donest play chords yet. But thank you anyway

**Comment by boris-mandrovnyj-krizik** [comment]
@OstbornTV I downloaded your exe main version. I have an issue with playing some of the sheets and laso there's an issue within simultaneously played notes. Mind helping me again?

**Comment by boris-mandrovnyj-krizik** [comment]
> My version is not intended to allow multiple notes to be played at the same time, so I can't really help with that as the version is only based on pressing one note at a time. I would link you to another project which allows multiple notes and which is also relatively good.
> 
> Edit: https://github.com/Viwyn/Sky-Music-Player

Thank you ;)

**Comment by boris-mandrovnyj-krizik** [comment]
> My version is not intended to allow multiple notes to be played at the same time, so I can't really help with that as the version is only based on pressing one note at a time. I would link you to another project which allows multiple notes and which is also relatively good.
> 
> Edit: https://github.com/Viwyn/Sky-Music-Player

Thank you for the recommendation but I did everythink in the Readme file yet pygetwindow doesnt want to work or Im just lost.

![Image](https://github.com/user-attachments/assets/be61925f-fefa-4398-be72-8e98da4590a9)

Here's a screenshot from terminal.

**Comment by bee4bee** [comment]
> To Whom It May Concern I am sorry to bother, but I am not able to make this work I can not make this program work. After some time trying to fix this program I can not get it to work. I found out this program can not know on which disk it is, so i wrote into the code relocation to my C:\ where python is located. Even after this the app can not run and has always an issue. I tried to even redirect it exactly to the folder where is python that I use for my personal funky stuff in VSC to run my crappy calculator app. It still will not launch it self and it even says that the file I installed does not exist. Yours Faithfully ![image](https://github.com/user-attachments/assets/5be26734-68c4-4214-ab1c-1b3515e6ab88) ![image](https://github.com/user-attachments/assets/79a8e25c-47ca-4db2-8771-0aaf91214ea0) ![image](https://github.com/user-attachments/assets/d7b989c3-50e8-4b82-a551-22b487973552)

exactly the same problem :(
